services/submission/src/main/java/app/coronawarn/server/services/submission/controller/SubmissionController.java

```diff
@@ -60,25 +61,24 @@
 
   private final TanVerifier tanVerifier;
 
-  @Autowired
-  SubmissionController(DiagnosisKeyService diagnosisKeyService, TanVerifier tanVerifier) {
-    this.diagnosisKeyService = diagnosisKeyService;
-    this.tanVerifier = tanVerifier;
-  }
+  private final Double fakeDelayMovingAverageSamples;
 
-  // Exponential moving average of the last N real request durations (in ms), where
-  // N = fakeDelayMovingAverageSamples.
-  @Value("${services.submission.initial_fake_delay_milliseconds}")
-  private Double fakeDelay;
+  private final Integer retentionDays;
 
-  @Value("${services.submission.fake_delay_moving_average_samples}")
-  private Double fakeDelayMovingAverageSamples;
+  private final Integer maxNumberOfKeys;
 
-  @Value("${services.submission.retention-days}")
-  private Integer retentionDays;
+  private Double fakeDelay;
 
-  @Value("${services.submission.payload.max-number-of-keys}")
-  private Integer maxNumberOfKeys;
+  @Autowired
```

Comment:
As mentioned before, `@Autowired` is not required at constructors when there is only one constructor.